packages/neon/neon/lib/src/settings/models/nextcloud_app_options.dart

```diff
@@ -1,23 +1,66 @@
+import 'package:meta/meta.dart';
+import 'package:neon/src/settings/models/exportable.dart';
 import 'package:neon/src/settings/models/option.dart';
 import 'package:neon/src/settings/models/options_category.dart';
 import 'package:neon/src/settings/models/storage.dart';
 
-abstract class NextcloudAppOptions {
-  NextcloudAppOptions(this.storage);
+/// Collection of [Option]s.
+abstract class OptionsCollection implements Exportable {
+  /// Creates a new OptionsCollection.
```

Comment:
Not really helpful, I usually just ignore constructors

Reply:
Should I remove them?

Reply:
Yes please